hack/test/e2e.sh

```diff
@@ -198,25 +199,25 @@ function get_kubeconfig {
 }
 
 function dump_cluster_state {
-  nodes=$(${KUBECTL} get nodes -o jsonpath="{.items[*].status.addresses[?(@.type == 'InternalIP')].address}" | tr [:space:] ',')
-  "${TALOSCTL}" -n ${nodes} services
+  nodes=$(${KUBECTL} get nodes -o jsonpath="{.items[*].status.addresses[?(@.type == 'InternalIP')].address}" | tr '[:space:]' ',')
+  "${TALOSCTL}" -n "${nodes}" services
   ${KUBECTL} get nodes -o wide
   ${KUBECTL} get pods --all-namespaces -o wide
 }
 
 function build_registry_mirrors {
   if [[ "${CI:-false}" == "true" ]]; then
-    REGISTRY_MIRROR_FLAGS=
+    REGISTRY_MIRROR_FLAGS=()
 
     for registry in docker.io registry.k8s.io quay.io gcr.io ghcr.io registry.dev.talos-systems.io; do
       local service="registry-${registry//./-}.ci.svc"
-      local addr=`python3 -c "import socket; print(socket.gethostbyname('${service}'))"`
+      addr=$(python3 -c "import socket; print(socket.gethostbyname('${service}'))")
```

Comment:
Why drop `local`?

Reply:
https://www.shellcheck.net/wiki/SC2155

Reply:
Maybe declare it separately then?

Reply:
that didn't matter much here :sweat_smile: